test/unit/core/metafunction/range_iterator_test.cpp

```diff
@@ -58,7 +58,7 @@ void expect_same_types()
     constexpr bool val = std::is_same_v<meta::at_c<list1, pos>, meta::at_c<list2, pos>>;
     if constexpr (!val)
     {
-        std::cerr << "\'" << detail::get_display_name_v<meta::at_c<list1, pos>> << "\' not equal to \'"
+        std::cerr << "pos: " << pos << " \'" << detail::get_display_name_v<meta::at_c<list1, pos>>.str() << "\' not equal to \'"
```

Comment:
if you we keep your small_string overload for ostream introduced in the former commit you should not need the `.str()` here.